### ISSUE TITLE
Lossy model, great top-1.

### ISSUE THREAD

**Issue by CEDAclaire** [NVIDIA/DIGITS#1587]
This are the model results from a faulty gear dataset. The vibration measurements were represented as spectrograms, which were transformed in DIGITS using the fill option. After training the model, I took the test.txt file and ran it through the Classify Many Images option, and the top-1 results were less than the model accuracy, even thought the model has nearly no loss.
![image](https://cloud.githubusercontent.com/assets/25851927/25146870/82e7aebe-2443-11e7-832e-5d639aeecb51.png)

Here, the only difference is that the dataset transformation was squashed. This model is less accurate and far more lossy (validation loss is 30x greater), yet the top-1 results of the associated test.txt file are impeccable! 
![image](https://cloud.githubusercontent.com/assets/25851927/25146889/94f45a30-2443-11e7-9e60-a0532a22911f.png)

Running the 'filled' dataset past 40 epochs causes overtraining and poorer top-1 results, so running both models to 60 epochs will not help.
I do not believe that this is a bug, but I would appreciate any explanation.
